src/core/ledger/policies/immediate.js

```diff
@@ -22,16 +22,33 @@ export type ImmediatePolicy = {|
 
 /**
  * Split a grain budget in proportion to the cred scores in
- * the most recent time interval.
+ * the most recent time interval, with the option to extend the interval
+ * to include the last {credIntervalCount} weeks.
  */
 export function immediateReceipts(
   budget: G.Grain,
-  identities: ProcessedIdentities
+  identities: ProcessedIdentities,
+  credIntervalCount: number = 1
```

Comment:
Why isn't the `credIntervalCount` part of the policy? Seems like good information to include there e.g. for future audit-ability. In whihc case this function can just take the `ImmediatePolicy` object, in addition to the identities.